src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,213 @@
+
+#### Hardware information
+
+<!-- This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("eyetrack.EyeTrackingHardware") }}
+
+### Example sidecar JSON document
+
+Below is an example of the sidecar JSON files
+for a fMRI dataset with eye-tracking data.
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example(
+   {
+   "sub-01": {
+      "func": {
+         "sub-01_task-visualSearch_bold.nii.gz": "",
+         "sub-01_task-visualSearch_bold.json": "",
+         "sub-01_task-visualSearch_events.tsv": "",
+         "sub-01_task-visualSearch_events.json": "",
+         "sub-01_task-visualSearch_eyetrack.tsv": "",
+         "sub-01_task-visualSearch_eyetrack.json": "",
+         },
+      }
+   }
+) }}
+
+Content of `sub-01_task-visualSearch_eyetrack.json`:
+
+```JSON
+{
+    "Manufacturer": "SR-Research",
+    "ManufacturersModelName": "EYELINK II CL v4.56 Aug 18 2010",
+    "DeviceSerialNumber": "17535483",
+    "SoftwareVersion": "SREB1.10.1630 WIN32 LID:F2AE011 Mod:2017.04.21 15:19 CEST",
+    "SamplingFrequency": 1000,
+    "SampleCoordinateUnits": "pixel",
+    "SampleCoordinateSystem": "gaze-on-screen",
+    "EnvironmentCoordinates": "top-left",
+    "RecordedEye": "Both",
+    "ScreenAOIDefinition": ["square",
+                            [100, 150, 300, 350]]
+}
+```
+
+Content of `sub-01_task-VisualSearch_events.json`:
+
+```JSON
+{
+   "TaskName": "Visual Search",
+   "InstitutionName": "Stanford University",
+   "InstitutionAddress": "450 Serra Mall, Stanford, CA 94305-2004, USA",
+   "StimulusPresentation": {
+       "ScreenDistance": 60,
+       "ScreenRefreshRate": 60,
+       "ScreenResolution": [1024, 768],
+       "ScreenSize": [0.386, 0.29]
+   }
+}
+```
+
+### Example Datasets
+<!--
+1. Datasets will be updated later to adapt to the agreed format.
+2. We aim at adding a last example converting published dataset from openeuro.
+-->
+
+-   Combined behavior and eye-tracking fixation and saccade data,
+    measured with an Eyelink (SR Research), from 8 particpants reading 320
+    embedded target words and invisible boundary (see
+    [Gagl, 2016](https://peerj.com/articles/2467/)).
+
+    [BIDS dataset](https://zenodo.org/record/1228659)
+
+-   Combined behavior and eye-tracking position and pupil data, measured with
+    an Eyelink (SR Research), from 26 participants performing a
+    binocular rivalry task (see
+    [Brascamp et.al, 2021](https://doi.org/10.7554/eLife.66161)).
+
+    [BIDS dataset](https://doi.org/10.5061/dryad.41ns1rncp)
```

Comment:
Hi - quick question: This link leads me to a page with `behavioral.zip` and `eye.zip` files. Upon downloading and unpacking, the files are not organized in BIDS format. I was wondering if this is an error or I'm missing something. Just wanted to bring this to your attention. thanks!

Reply:
Thanks for the ping. We are currently working on converting those datasets, so at the moment the link point to the source data (non bidsified).